server/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "restApiRoot": "/api",
   "host": "0.0.0.0",
-  "port": 3000,
+  "port": 54321,
```

Comment:
@nlkluth is this just arbitrary?

Reply:
@jsullivan5 this file is configuring the port we are listening for the loopback server. My guess is that it was just changed to an arbitrary number since 3000 is the default for create-react-app (the new front end setup).

Reply:
Yeah - Can switch it to whatever though!